docs/release-notes/set-log-retention.rst

```diff
@@ -0,0 +1,17 @@
+:orphan:
+
+**New Features**
+
+-  CLI: Two new CLI commands have been added, ``det e set log-retention <exp-id>`` and ``det t set
+   log-retention <trial-id>``. These commands allow the user to set the log retention days for
+   experiments and trials. These commands take the required argument ``--days`` and the optional
+   argument ``forever`. ``--days`` sets the number of days to retain the logs for, from the time of
+   creation and ``--forever`` retains logs forever. Set ``--days`` to ``null`` or ``none`` when
+   using ``--forever``.
+
+-  WEBUI: Users can now Retain Logs for multiple experiments by selecting a few experiments from the
+   Experiment List page and clicking "Retain Logs" from the Actions dropdown. Then, they will be
+   able to input the number of days they want to set log retention to, or select the "Forever"
+   checkbox to retain them forever. A new columns has also been added to the Trial List page called
+   "Log Retention Days". This column will show the amount of time the logs for each trial will be
+   retained for, from the time of creation.
```

Comment:
-  WebUI: Add support for retaining logs for multiple experiments by selecting experiments from the Experiment List page and choosing **Retain Logs** from **Actions**. Users can then input the desired number of days for log retention or select the "Forever" checkbox for indefinite log retention. Add a  new column, ''Log Retention Days'', to the Trial List page. ''Log Retention Days'' displays the duration logs will be retained for each trial from creation.